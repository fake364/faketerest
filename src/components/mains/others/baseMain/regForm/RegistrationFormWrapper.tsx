import React from 'react';
import { useTranslation } from 'next-i18next';
import RegForm from './form/RegForm';

type Props = {};

const RegistrationFormWrapper: React.FC<Props> = () => {
  const { t } = useTranslation('main-page');

  return (
    <div className="main-reg-form">
      <h1 className="text-[70px] text-white max-w-[600px]">{t('regForm.header')}</h1>
      <RegForm />
    </div>
  );
};

export default RegistrationFormWrapper;
