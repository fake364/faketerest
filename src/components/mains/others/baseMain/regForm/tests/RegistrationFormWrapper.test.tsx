import React from 'react';
import { shallow } from 'enzyme';
import RegistrationFormWrapper from '../RegistrationFormWrapper';

describe('RegistrationFormWrapper', () => {
  it('should render RegForm with mode switcher', () => {
    const wrapper = shallow(<RegistrationFormWrapper />);
    expect(wrapper).toMatchSnapshot();
  });
});
