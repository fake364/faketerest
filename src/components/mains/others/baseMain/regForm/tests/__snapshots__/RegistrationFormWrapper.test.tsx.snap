// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationFormWrapper should render RegForm with mode switcher 1`] = `
<div
  className="main-reg-form"
>
  <h1
    className="text-[70px] text-white max-w-[600px]"
  >
    regForm.header
  </h1>
  <RegForm
    isRegistrationShown={true}
    setRegistrationShown={[Function]}
  />
</div>
`;
